src/cli_driver.rs

```diff
@@ -363,7 +363,7 @@ struct ProcessingSession {
     /// This is the virtual "CWD" that our filesystem accesses use. It is the
     /// dirname of `primary_input_path`, or an empty path (i.e., corresponding
     /// to the CWD if `primary_input_path` is None.
-    fs_root: PathBuf,
+    #[allow(unused)] fs_root: PathBuf,
```

Comment:
I put this in because now it is unused. But maybe it will be needed later?
Else there would be a warning while compiling.